src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,23 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int OLDER_THAN_AGE = 35;
+    private static final String NATIONALITY = "Ukrainian";
+    private static final String REGEX = "-";
+    private static final int START_PERIOD = 0;
+    private static final int END_PERIOD = 1;
```

Comment:
```suggestion
    private static final int START_PERIOD_INDEX = 0;
    private static final int END_PERIOD_INDEX = 1;
```
A bit better (its up to you)